magefile.go

```diff
@@ -659,7 +662,7 @@ func (t Test) IntegrationCIPrepare(ctx context.Context) error {
 // to the addon operator CSV present in the cluster.
 func (t Test) IntegrationCIInjectEnvVariable(ctx context.Context) error {
 	cluster, err := dev.NewCluster(path.Join(cacheDir, "ci"),
-		dev.WithKubeconfigPath(os.Getenv("KUBECONFIG")), dev.WithSchemeBuilder(operatorsv1alpha1.SchemeBuilder))
+		dev.WithKubeconfigPath(os.Getenv("KUBECONFIG")), dev.WithSchemeBuilder(runtime.SchemeBuilder{operatorsv1alpha1.AddToScheme, aoapisv1alpha1.AddToScheme}))
```

Comment:
`aoapisv1alpha1.AddToScheme` is not required here too

Reply:
Gotcha, will do